awx/main/management/commands/custom_venvs.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2015 Ansible, Inc.
+# All Rights Reserved
+
+import json
+
+from awx.main.utils.common import get_custom_venv_choices, get_custom_venv_pip_freeze
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    """Returns either a list of custom venv paths or outputs the pip freeze from the path passed in the argument"""
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--path',
+            dest='path',
+            type=str,
+            default='',
+            help='run without arguments to see a list of paths, run with one of those paths as an argument and see the pip freeze data',
```

Comment:
As a matter of wording:

> run without arguments to see a list of paths

I think that should be left for the general command help text to establish (as opposed to the help text about the option).

Will this take either an absolute path or a relative path?